azerothcore/azerothcore-wotlk
# fix(DB/smart_scripts): gossip texts and SAI for quest Get the Scoop (1950)

## Changes Proposed:
-  ported from https://github.com/TrinityCore/TrinityCore/issues/21956
-  added SAI and related gossip menu links

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/5857
- Closes https://github.com/chromiecraft/chromiecraft/issues/651

## SOURCE:


## Tests Performed:

- tested in-game


## How to Test the Changes:


1. create a mage
2. .q add 1950
3. go to thousand needles (.go cr id 6548)
4. go to the NPCs Goblin Pit Crewman, Gnome Pit Crewman, Race Master Kronkrider and you'll see the new gossip menu texts
5. go to "Plucky" Johnson and type /beckon, he will turn into a human and will tell you "a secret" via gossip menu and complete you the quest



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
